functorch/csrc/BatchingRegistrations.cpp

```diff
@@ -1372,8 +1415,8 @@ TORCH_LIBRARY_IMPL(aten, FT_BATCHED_KEY, m) {
   m.impl("slice.Tensor", slice_batching_rule);
   m.impl("split.Tensor", split_batching_rule);
   m.impl("split_with_sizes", split_with_sizes_batching_rule);
-  m.impl("squeeze", squeeze_batching_rule);
-  m.impl("squeeze.dim", squeeze_dim_batching_rule);
+  VMAP_SUPPORT("squeeze", squeeze_batching_rule);
+  VMAP_SUPPORT("squeeze.dim", squeeze_dim_batching_rule);
```

Comment:
Let's move these functions to `BatchRulesView.cpp` -- squeeze is a view function and BatchingRegistrations.cpp is meant for the old-style batching rules (it's not documented as so though)